src/components/components/App/App.js

```diff
@@ -0,0 +1,12 @@
+import Footer from '../Footer/Footer';
+import './App.scss';
+
+function App() {
+  return (
+    <div className="App">
+      <Footer/>
+    </div>
+  );
+}
+
+export default App;
```

Comment:
please revert this file to the original state (from main branch), as it would lead to merge conflicts otherwise